docs/plugins/subtitles.md

```diff
@@ -0,0 +1,63 @@
+# Subtitles
```

Comment:
Because the Subtitles are now integrated from within the VideoPlayer, I don't think they need their standalone docs